net-irc/znc/znc-9999.ebuild

```diff
@@ -2,163 +2,157 @@
 # Distributed under the terms of the GNU General Public License v2
 # $Id$
 
-EAPI=5
+EAPI=6
 
-PYTHON_COMPAT=( python3_4 )
-inherit autotools eutils python-single-r1 user
+PYTHON_COMPAT=( python{3_4,3_5} )
+PLOCALES="ru"
+CMAKE_MAKEFILE_GENERATOR=ninja
```

Comment:
Well, not always if the user opts out (user can disable cleaning, ignore/bypass ebuild phase locks) but by default this isn't a problem and we don't care.